opwen_email_server/services/auth.py

```diff
@@ -188,3 +188,11 @@ def _domain_file(cls, domain: str) -> str:
     @classmethod
     def _client_id_file(cls, client_id: str) -> str:
         return f'client_id/{client_id}'
+
+
+class NoAuth(AzureAuth):
```

Comment:
Given that `NoAuth` always returns `True`, what do we gain from inheriting from `AzureAuth`? I'd recommend to instead implement the same "interface" (i.e., the `is_owner` and `domain_for` methods) and not passing in the storage instance (this will prevent people from accidentally using the storage in the future).

Reply:
Doesn't gain much at all. I was just worried about the type signature for the auth in [ReceiveInboundEmail](https://github.com/ascoderu/opwen-cloudserver/blob/ef72c504c29e3776741e587b4e62041dc69caa90/opwen_email_server/actions.py#L215).